clang/include/clang/Serialization/ASTBitCodes.h

```diff
@@ -255,6 +255,12 @@ class DeclOffset {
   }
 };
 
+// The unaligned decl ID used in the Blobs of bistreams.
+using unalighed_decl_id_t =
```

Comment:
I know this typo isn't introduced by your PR, but since your PR is touching it, please fix it in a prep commit.
```suggestion
using unaligned_decl_id_t =
```

Reply:
Nice catch! Done in https://github.com/llvm/llvm-project/commit/e73e4951b20c70f24354e2a2820876c818dcaee3